string-manipulation-2/src/ransom-case.js

```diff
@@ -0,0 +1,17 @@
+/* exported ransomCase */
+
+function ransomCase(string) {
+  var ransomCaseResult = '';
+  var lowCase = '';
+  var upCase = '';
+  for (var i = 0; i < string.length; i++) {
+    if (i % 2 === 0) {
+      lowCase = string.substring(i, i + 1);
+      ransomCaseResult += lowCase.toLowerCase();
+    } else {
+      upCase = string.substring(i, i + 1);
+      ransomCaseResult += upCase.toUpperCase();
+    }
```

Comment:
See if you can simplify this a bit by avoiding slicing your strings. You can index into a string like you would an array.